analytic_engine/src/manifest/details.rs

```diff
@@ -253,11 +265,19 @@ where
         }
 
         if has_logs {
+            debug!(
+                "Manifest recover with only logs, table_id:{}, space_id:{}",
+                self.table_id, self.space_id
+            );
```

Comment:
Maybe we can just print one log instead of L268 and L277:
```
            debug!(
                "Manifest finish recover without prev snapshot, has_logs:{has_logs}, table_id:{}, space_id:{}",
                self.table_id, self.space_id
            );
```